script.js

```diff
@@ -97,14 +98,14 @@ function removeEventListeners(){
     }
 }
 
-function gameOver(win){
+function gameOver({ line, winner }){
```

Comment:
we can destructure the object win, this will make us avoid typed `win.property` each time